### ISSUE TITLE
add shaed-mime-info to pcmanfm-qt

### ISSUE THREAD

**Issue by wamserma** [NixOS/nixpkgs#76901]
**Describe the bug**
No default file associations are given in pcmanfm-qt, the default file manager in the lxqt desktop package, e.g. when i click on a USB drive in the status bar, its root folder is opened in VS Code instead of pcmanfm-qt.

Adding `shared-mime-info`to https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/lxqt/pcmanfm-qt/default.nix like here https://github.com/NixOS/nixpkgs/pull/35458/files or here https://github.com/NixOS/nixpkgs/pull/14217/files might help as it did for https://github.com/NixOS/nixpkgs/issues/12280

Other related PRs:
https://github.com/NixOS/nixpkgs/pull/58203  

**To Reproduce**
1. Install NixOS 19.09 with SDDM and LXQT desktop
2. add (unfree) VS Code to configuration.nix and rebuilt
3. run `xdg-mime query default inode/directory`and observe output: `code.desktop`

**Expected behavior**
`xdg-mime query default inode/directory`should return `pcmanfm-qt.desktop` when defaults are used.

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.
 - host os: `Linux 4.19.92, NixOS, 19.09.1748.ad1e1af5ad3 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"nixos-19.09.1748.ad1e1af5ad3, unstable-20.03pre194293.2436c27541b"`
 - channels(user): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: pcmanfm-qt lxqt
# a list of nixos modules affected by the problem
module:
```

**Comment by romildo** [comment]
Maybe this behavior comes from the fact that `vscode` is also associated with the `inode/directory` mime type:

```
$ grep MimeType /nix/store/ha1qx43rzpn1ccz30vr6v2iwj0jrr505-vscode-1.41.1/share/applications/code.desktop
MimeType=text/plain;inode/directory;
```

Than it is up to the user to choose his preferred application for opening directories.

**Comment by wamserma** [comment]
Yes. But when multiple applications are associated with the same mime type, the associations are currently somewhat arbitary. I see a similiar effect with `.mp3`-Files: When `nixos-rebuild switch --upgrades` brings in a fresh Firefox, `.mp3`s open in FF instead of VLC until next reboot. But having folders opening in `vscode` is the most annoying issue.
Maybe this issue is a more general LXQT-Desktop thing.

**Comment by romildo** [comment]
@worldofpeace What do you think about this issue? Is there a recommended way of setting a default application system-wide?

**Comment by wamserma** [comment]
This partly seems to be an upstream issue: https://github.com/lxqt/lxqt/issues/652
`.mp3`s opened with `xdg-open` from the terminal go to VLC, opened from gPodder, they open in Firefox (not permanently, but consistently between reboots). Funnily, all settings/mimeinfo-files I checked point to VLC.
